src/AspNetCoreModuleV2/CommonLib/iapplication.h

```diff
@@ -11,6 +11,7 @@ enum APPLICATION_STATUS
     STARTING,
     RUNNING,
     SHUTDOWN,
+    OFFLINE,
```

Comment:
Can we rename this APPOFFLINE_REMOVED?

Reply:
Not to `APPOFFLINE_REMOVED` it's not about app_offline, just that `IAPPLICATION` instance should be replaced with a new one. `STOPPED`?

Reply:
Sure.

Reply:
Or maybe `RECYCLED`